kafka-init/scripts/kafka-init-run.template.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Sorry for the dump question but how and where this "template" file is used? I see the same for topic and user operator but then the created init container use a "plain" kafka-initi-run.sh (the same for topic and user operators).

Reply:
It's interpolated via `envsubst` run from the `Makefile`, but don't worry about the details: I had out-of-band conversation with Jakub and Kyle yesterday and we're going to change how this stuff works. In short maven will create a zip archive including the jars and scripts for each operator and these will then be unzipped/combined into a single operator image. It will be slightly neater. I should be able to update the PR later today.

Reply:
Out of curiosity are you going to use the `ADD` of the zip in the Dockerfile which unzip jars and scripts automatically?

Reply:
No. The `Makefile` is doing the unzipping and the `Dockerfile` is then `COPY`ing the resulting directories. I don't suppose it makes a difference as I think both ways add two layers to the image.